### ISSUE TITLE
Support for iOS

### ISSUE THREAD

**Issue by usmaanirfan** [obgm/libcoap#543]
Can I use this library for iOS?

Thanks,
Usman

**Comment by obgm** [comment]
I do not think so. I have no experience with IOS but I doubt that it provides the common POSIX networking interface required by libcoap. There are Android ports of libcoap, though.
